### ISSUE TITLE
Is there a Roadmap?

### ISSUE THREAD

**Issue by gtamas** [bahmutov/cypress-vue-unit-test#400]
I'm a bit confused about this. I can see this is still alpha, but many features are already implemented and it seems to be usable for most purposes.

About when can we expect the first "stable" version (1.0) to be released? Could you give me just a rough estimate?  Or is there a roadmap page that I could check somewhere?

We would like to use this kinda testing in the real world project which will go to prod in a few months. Is this already stable enough to be used in such a project, or should we wait for 1.0?  

Actually normal e2e testing with Cypress also fits our needs, but this would be obviously much better.  

Thanks in advance!

**Comment by bahmutov** [comment]
@gtamas great question. To me, if your tests work with this component testing approach, then all is good (I know bundling is slow, we are working on it), and it is stable.

**Comment by JessicaSachs** [comment]
@gtamas You can use the mount functions as documented with the latest alpha. We're deeming these the "mount utils". As the vuejs/test-utils maintainer, I intend to keep the APIs very very similar (we mount using VTU under the hood), but we may still have a breaking change or two.

Vue 3 will likely be a major version bump as well (since VTU is also doing a major version bump)

Areas to be careful around:
Mixing support.js files and plugins.js files from E2E and component if you're trying to have a project with both. Essentially: try not to spin up your database in a beforeAll for your component tests :P 

Timing for performance solutions: We're getting our ducks in order about performance and considering releasing a standalone dev server solution to mitigate Cypress's need for shipping such a large desktop app. This is just spinning up and will be out in a few months.

**Comment by santicros** [comment]
Thanks for your work!! One question I have is are you thinking of supporting Vite at some point?

**Comment by JessicaSachs** [comment]
Yes, but it won’t be immediate. We’re moving to a dev server based approach partly to support more than just webpack
